lib/internal/modules/esm/default_loader.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const { defaultLoad } = require('internal/modules/esm/load');
+const { defaultResolve } = require('internal/modules/esm/resolve');
+
+exports.resolve = defaultResolve;
+exports.load = defaultLoad;
```

Comment:
If we create a new file we might need to add it to the startup snapshot (see https://github.com/nodejs/node/pull/45849). Alternatively is there a way to stick this in an existing file?

Reply:
The reason I put this in its own file is so that the `url` property actually led to a valid file that represented the base "loader". 

It can live in literally any file that can be treated as a loader (exports `load`, `resolve`, etc.) This code is the relevant block:

```js
const defaultLoader = 'internal/modules/esm/default_loader';
this.addCustomLoader(`node:${defaultLoader}`, require(defaultLoader));
```

So that if we ever expose a list loaders or anything like that the `url` property is what you see above.